ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywallView.kt

```diff
@@ -63,12 +64,15 @@ private fun getPaywallViewModel(
     offering: Offering?,
     listener: PaywallViewListener?,
 ): PaywallViewModel {
+    val applicationContext = LocalContext.current.applicationContext
     return viewModel<PaywallViewModelImpl>(
         factory = PaywallViewModelFactory(
-            LocalContext.current.applicationContext.toAndroidContext(),
+            applicationContext.toAndroidContext(),
             mode,
             offering,
             listener,
+            applicationContext.packageName,
```

Comment:
Yeah we're already passing `ApplicationContext` here, so let's add a method to that interface instead?

Reply:
Yes, let's do that, much cleaner